sdk/storage/azure-storage-blob-changefeed/src/main/java/com/azure/storage/blob/changefeed/BlobChangefeedAsyncClient.java

```diff
@@ -4,10 +4,38 @@
 package com.azure.storage.blob.changefeed;
 
 import com.azure.core.annotation.ServiceClient;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.storage.blob.BlobContainerAsyncClient;
+import com.azure.storage.blob.BlobContainerClientBuilder;
+import com.azure.storage.blob.BlobServiceVersion;
 
 /**
- * TEMP
+ * This class provides a client that contains all operations that apply to Azure Storage Blob changefeed.
```

Comment:
I'd like to have more informative docs. Not sure if you were planning on adding that later.

Reply:
yep, I will add more information as I understand more about ChangeFeed and add more related classes.